ui/src/app/admin/_components/personTable.tsx

```diff
@@ -0,0 +1,284 @@
+import { groupingOwners, removeFromGroups } from '@/lib/actions';
+import OwnersModal from '@/components/modal/owners-modal';
+import { Alert, AlertDescription } from '@/components/ui/alert';
+
+const pageSize = parseInt(process.env.NEXT_PUBLIC_PAGE_SIZE as string);
+
+const PersonTable = (data) => {
+    const [globalFilter, setGlobalFilter] = useState('');
+    const [sorting, setSorting] = useState<SortingState>([]);
+    const [rowSelection, setRowSelection] = useState<RowSelectionState>({});
+    const [modalOpen, setModalOpen] = useState(false);
+    const [modalData, setModalData] = useState([]);
+    const [dummyBool, setDummyBool] = useState(false);
+    const router = useRouter();
+    const searchUid = data.searchUid;
+    const validUid = data.groupingsInfo.resultCode;
+    const groupingsInfo = data.groupingsInfo.results;
+    const userInfo = data.userInfo;
+    const hydrateModal = async (path) => {
+        setModalData((await groupingOwners(path)).members);
+        setModalOpen(true);
+    };
+
+    const close = () => {
+        setModalOpen(false);
+    };
+
+    const handleRemove = async () => {
+        const numSelected = table.getSelectedRowModel().rows.length;
+        const arr = [];
+        let i;
+        for (i = 0; i < numSelected; i++) {
+            const original = table.getSelectedRowModel().rows[i].original;
+            if (original.inOwner) arr.push(original.path + ':owners');
+            if (original.inInclude) arr.push(original.path + ':include');
+            if (original.inExclude) arr.push(original.path + ':exclude');
+        }
+        await removeFromGroups(searchUid, arr);
+        setRowSelection({});
+        router.refresh();
+    };
+
+    const table = useReactTable({
+        columns: personTableColumns,
+        data: groupingsInfo,
+        getCoreRowModel: getCoreRowModel(),
+        getPaginationRowModel: getPaginationRowModel(),
+        getFilteredRowModel: getFilteredRowModel(),
+        getSortedRowModel: getSortedRowModel(),
+        state: { globalFilter, sorting, rowSelection },
+        initialState: { pagination: { pageSize } },
+        onGlobalFilterChange: setGlobalFilter,
+        onSortingChange: setSorting,
+        onRowSelectionChange: setRowSelection
+    });
+
+    return (
+        <>
+            <OwnersModal open={modalOpen} close={close} modalData={modalData} />
+            <div className="flex flex-col md:flex-row md:justify-between pt-1 mb-1">
+                <div className="flex items-center">
+                    <h1 className="text-[2rem] font-medium text-text-color md:justify-start pt-3">Manage Person</h1>
+                    <p className="text-[1.2rem] font-medium text-text-color md:justify-end pt-5 ps-2">
+                        {userInfo === undefined
+                            ? ''
+                            : '(' + userInfo.name + ', ' + userInfo.uid + ', ' + userInfo.uhUuid + ')'}
+                    </p>
```

Comment:
Instead of the ternary operator, it's nicer to use the && operator:
```
{userInfo !== undefined && <p className="text-[1.2rem] font-medium text-text-color md:justify-end pt-5 ps-2">
    {'(' + userInfo.name + ', ' + userInfo.uid + ', ' + userInfo.uhUuid + ')'}
</p>}
```